src/playground-ide.ts

```diff
@@ -147,17 +201,67 @@ export class PlaygroundIde extends LitElement {
       >
       </playground-file-editor>
 
-      <playground-preview
-        part="preview"
-        exportparts="preview-toolbar,
-                     preview-location,
-                     preview-reload-button,
-                     preview-loading-indicator"
-        location="Result"
-        .project=${projectId}
-      ></playground-preview>
+      <div id="rhs">
+        ${this.resizable
+          ? html`<div
+                id="resizeBar"
+                @mousedown=${this.onResizeBarMousedown}
+              ></div>
+              <div id="resizeOverlay"></div>`
+          : nothing}
+
+        <playground-preview
+          part="preview"
+          exportparts="preview-toolbar,
+                       preview-location,
+                       preview-reload-button,
+                       preview-loading-indicator"
+          location="Result"
+          .project=${projectId}
+        ></playground-preview>
+      </div>
     `;
   }
+
+  async update(changedProperties: PropertyValues<this>) {
+    super.update(changedProperties);
+    if (changedProperties.has('resizable') && this.resizable === false) {
```

Comment:
this is a slightly pedantic suggestion, but generally most work inside `update` IMO should go before `super.update` as to accidentally prevent a rerender cycle. Pedantic because this conditional doesn't actually cause a rerender, but future work that may trigger render cycle, people may just add onto this.

Reply:
Done.